subprojects/common/elastic/src/main/kotlin/com/avito/android/elastic/DateFormatCheckerImpl.kt

```diff
@@ -0,0 +1,21 @@
+package com.avito.android.elastic
+
+import java.util.Locale
+
+internal class DateFormatCheckerImpl : DateFormatChecker {
```

Comment:
```suggestion
internal class ElasticDateFormatChecker : DateFormatChecker {
```
`Impl` -> `Elastic` to express the purpose